src/js/messaging/components/buttons/ButtonDelete.jsx

```diff
@@ -12,24 +11,19 @@ class ButtonDelete extends React.Component {
   }
 
   render() {
-    const isTextVisible = classNames({
-      'usa-sr-only': this.props.compact
-    });
-
     return (
       <button
           onClick={this.handleClick}
-          className="messaging-btn-delete"
+          className="va-icon-link messaging-btn-delete"
```

Comment:
If the other buttons have a `msg-btn-*` class, it would be more consistent to make this the same prefix.